package.json

```diff
@@ -8,12 +8,10 @@
     "fix": "eslint . --fix && prettier . -w",
     "lint": "eslint . && prettier . -c",
     "lockfile-lint": "lockfile-lint --allowed-hosts npm --allowed-schemes https: --empty-hostname false --type npm --path package-lock.json",
-    "remark": "npm run remark-local && npm run remark-ok && npm run remark-fail && npm run remark-change && npm run remark-node-core",
-    "remark-change": "remark test/input.md --use ./index.js | diff test/output.md -",
+    "remark": "npm run remark-unit && npm run remark-local && npm run remark-node-core",
     "remark-local": "remark *.md --use ./index.js -fq",
```

Comment:
Wonder if these should be renamed too. Maybe `test:local-config`, `test:node-core`, etc...

Reply:
Done.